public/app/percona/add-instance/components/AddRemoteInstance/FormParts/HAProxyConnectionDetails/HAProxyConnectionDetails.test.tsx

```diff
@@ -1,19 +1,18 @@
-import { mount } from 'enzyme';
 import React from 'react';
 import { Form } from 'react-final-form';
-import { dataTestId } from '@percona/platform-core';
+import { fireEvent, render, screen } from '@testing-library/react';
 import { HAProxyConnectionDetails } from './HAProxyConnectionDetails';
 
-xdescribe('HAProxy connection details:: ', () => {
+describe('HAProxy connection details:: ', () => {
   it('should trim username and password values right', async () => {
-    const root = mount(
-      <Form onSubmit={jest.fn()} render={() => <HAProxyConnectionDetails remoteInstanceCredentials={{}} />} />
-    );
+    render(<Form onSubmit={jest.fn()} render={() => <HAProxyConnectionDetails remoteInstanceCredentials={{}} />} />);
 
-    root.find(dataTestId('username-text-input')).simulate('change', { target: { value: '    test    ' } });
-    root.find(dataTestId('password-password-input')).simulate('change', { target: { value: '    test    ' } });
+    const userNameTextInput = await screen.getByTestId('username-text-input');
```

Comment:
No need to await when using get as it doesn't return a promise, also this test doesn't need to be async anymore

Reply:
Yes, I think that at this point you are definitely right, I will review the tests again. Probably here I forgot to delete async after fixing find on get.